docs/src/atopics.md

```diff
@@ -51,6 +51,35 @@ f
 
 An alternative to this approach is to compute gradient quantities at samples points and plot these via `arrows`.
 
+### High-order fields
+
+The investigation of high-order fields is currently only supported via a first-order refinment of the
+problem.
+Here the high-order approximation is replaced by a first order approximation of the field, which is
+spanned by the nodes of the high-order approximation. For example, the first order refinement of a
+heat problem on a square domain for Lagrange polynomials of order 5 looks like this:
+```@example 1
+include("ferrite-examples/heat-equation.jl"); #defines manufactured_heat_problem
+
+f = GLMakie.Figure()
+axs = [GLMakie.Axis3(f[1, 1], title="Coarse"), GLMakie.Axis3(f[1, 2], title="Fine")]
+
+dh,u = manufactured_heat_problem(Triangle, Lagrange{2,RefTetrahedron,5}(), 1)
+dh_for,u_for = for_discretization(dh, u)
+plotter_for = MakiePlotter(dh_for, u_for)
+FerriteViz.surface!(axs[1], plotter_for)
+
+dh,u = manufactured_heat_problem(Triangle, Lagrange{2,RefTetrahedron,5}(), 3)
+dh_for,u_for = for_discretization(dh, u)
+plotter_for = MakiePlotter(dh_for, u_for)
+FerriteViz.surface!(axs[2], plotter_for)
+
+f
+```
```

Comment:
![image](https://user-images.githubusercontent.com/27497290/219015919-8fb491be-f55a-4c6d-ae0e-302b40a393bc.png)
something broken in the documenter deploy for this plot

Reply:
Derp.

Reply:
Fixed.